tests/integration/docker/info.bats

```diff
@@ -31,6 +31,6 @@ setup() {
 @test "Container info" {
 	container=$(random_name)
 	$DOCKER_EXE run -itd --name $container busybox
-	$DOCKER_EXE info| grep "Containers: 1"
+	$DOCKER_EXE info| grep "^ Running: 1"
```

Comment:
I was thinking the same, but I don't think we can reliably test this since `info` only gives a count so it's all going to be horribly racy.